bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PerChannelBookieClient.java

```diff
@@ -2488,6 +2490,14 @@ public void operationComplete(ChannelFuture future) throws Exception {
 
             makeWritable();
         }
+
+        private void logBookieUnavailable(Consumer<Void> log) {
+            final long now = System.currentTimeMillis();
+            if ((now - lastBookieUnavailableLog) > 30_000) {
```

Comment:
we should make this 30_000 configurable or calculate it from other timeout configuration parameters